addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/station/NAMainHandler.java

```diff
@@ -97,9 +98,11 @@ protected State getNAThingProperty(String channelId) {
                             dashboardData.getHumidity());
                     return toDecimalType(WeatherUtils.getDewPointDep(dashboardData.getTemperature(), dewpoint));
                 case CHANNEL_WIND_UNIT:
-                    return new DecimalType(userAdministrative.getWindunit());
+                    return userAdministrative != null ? new DecimalType(userAdministrative.getWindunit())
```

Comment:
Looking at the rest of the binding, it would seem that this belongs in the `ChannelTypeUtils`

Reply:
Sorry, I do not get your comment ..?

Reply:
But yes, I think it's worth having a test done by someone who faces the issue.

Reply:
Sorry for not being clear the rest of all null handling for value is handled inside the `ChannelTypeUtils` this one looks a little bit misplaced compare with line 99 for example.

Reply:
Nope, I was wrong... :-)

Reply:
It was not `userAdministrative` but `userAdministrative.getPressureunit()` I will update it  :-)